src/snapred/backend/log/logger.py

```diff
@@ -2,50 +2,64 @@
 # Date : Time : Level : Module.Class : Machine/Node : Message
 import logging
 import socket
+
+from mantid.utils.logging import log_to_python
+
 from snapred.meta.Config import Config
 
-from mantid.kernel import ConfigService
+# Configure Mantid to send messages to Python
+log_to_python()
```

Comment:
I'd prefer if it remained autonomous for cleanliness sake.  What effect would calling this from main have?

Reply:
I don't recall if it will be run every time `.../log/logger.py` is imported, but it'll definitely be run the first time. If there are steps that have to be done before that, you'll have to reproduce them to be in a state where you can write tests that use this code.